waku/nwaku.go

```diff
@@ -965,54 +981,74 @@ func (n *WakuNode) PingPeer(ctx context.Context, peerInfo peer.AddrInfo) (time.D
 }
 
 func (n *WakuNode) Start() error {
-	wg := sync.WaitGroup{}
+	Debug("Starting %s", n.nodeName)
```

Comment:
Wow! Looks nice indeed :)
A super nitpick comment: maybe we could add the prefix `Log` to make it clearer that it is logging. Not a big deal though ;P
```suggestion
	LogDebug("Starting %s", n.nodeName)
```